c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -68,6 +90,47 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
   this.modal_.open({animation: true, size: 'lg', template: template});
 };
 
+
+/**
+ * @param {Date} dateStart
+ * @param {Date} dateEnd
+ * @suppress {checkTypes} it told (found scope | null, required -> scope) at compile()(this.scope)...
```

Comment:
to just 'get rid of', probably yes :P apparently it works, but if someone has a better solution, it will be welcome. 
I've also got a problem with `app.ViewDetailsController.prototype.displayDate` at `if (this.firstTime_)` - read the comment.